### ISSUE TITLE
Publish montly n-quad dumps

### ISSUE THREAD

**Issue by essepuntato** [essepuntato/opencitations#16]
In order to improve the accessibility and reuse of the data, it is important to provide also a dump of the whole corpus as an n-quad archive. This should be implemented in the process for creating a publishing the monthly dump.

[Suggested by @stain as one of the reviewer of our ISWC 2017 paper]
